convert-redis-to-gemfire/session-state/spring-session-boot-simple/README.md

```diff
@@ -25,20 +25,10 @@ implementation 'org.springframework.session:spring-session-data-redis'
 Replace them with these dependencies:
 
 ```groovy
-ext {
-	set('springGeodeVersion', "1.4.0")
-}
 dependencies {
-implementation 'org.springframework.geode:spring-geode-starter'
-implementation 'org.springframework.geode:spring-geode-starter-session'
+implementation 'com.vmware.gemfire:spring-boot-3.1-vmware-gemfire-10.0:1.0.0'
```

Comment:
Seemingly missing the Gemfire session dependency

Reply:
Added it here as well.